modal/functions.py

```diff
@@ -427,6 +427,8 @@ async def poll_outputs():
 # Wrapper type for api_pb2.FunctionStats
 @dataclass
 class FunctionStats:
+    """Simple data structure storing stats for a running function."""
```

Comment:
Without adding this we seem to get a default from dataclass which looks weird.

<img width="760" alt="image" src="https://user-images.githubusercontent.com/12058921/217321165-fec2612a-430f-457f-81d0-cd9532fc9923.png">